src/commands/git_secret_tell.sh

```diff
@@ -64,6 +64,21 @@ function tell {
     _abort "you must provide at least one email address."
   fi
 
+  # TODO: this block is duplicated in git_secret_killperson.sh and should be factored
```

Comment:
Should we refactor it now or you feel like it's not the time yet?

Reply:
I'll take another look and let you know @sobolevn

Reply:
factored.